### PR TITLE
badge

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- please review
-

### CODE REVIEW

**Review by Rajitha04 — APPROVED** [review]
Use var colors instead giving them directly. Rest looks good to merge!

File: badge.css [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+}
+.a{
+    justify-content: flex-end;
+    display: flex;
+    align-items: center;
+    margin-left: auto;
+}
+.a1{
+    padding: 0px 15px;
+    font-size: 25px;
```
Comment (HarshnaNHaswani) [inline badge.css:19]: what is a, a1? please use names that are understood easily

File: badge.css [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    grid-column: 2/4;
+    grid-row: 2/6;
+    padding: 20px;
+}
+*{
+    margin: 0;
+}
+.main p,h3,div{
+    padding-top: 20px;
+}
```
Comment (HarshnaNHaswani) [inline badge.css:61]: format file

File: badge.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+.side-bar div{
+    margin-top: 10px;
+}
+.main{
+    grid-column: 2/4;
+    grid-row: 2/6;
+    padding: 20px;
+}
+*{
```
Comment (HarshnaNHaswani) [inline badge.css:56]: keep resets separate

**Review by HarshnaNHaswani — COMMENTED** [review]
Please use variables for colors, and use responsive units.
Rest looks good to go

File: badge.css (hunk 1 above)
Comment (RaghavendraaReddy) [reply badge.css:19]: allirght

**Pull request merged**